### DISCUSSION TITLE
Implementing a Tokenizer class for all the text processing and G2P

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#836] [Ideas]
I plan to muster all the text processing in a `Tokenizer` class.

If you have comments, opinions about how, why, and what, feel free to share here 👍

**Comment by arif334** [comment]
@erogol Do you have any update on this? Also, I have a question, although not directly relevant to this topic. I want to implement the G2P converter, and normalizer for my own language, `Bangla` (also known as `Bengali`, abbreviated as `bn` or `bn-BD`), and share these with the coqui community. Where can I get the directions for doing these?

**Reply by erogol** [reply]
No timeline set. But there is the TokenizerAPI PR you can follow the progress.
